app/controllers/dashboards_controller.rb

```diff
@@ -15,5 +15,7 @@ def show
                       .where('reviewers.user_id' => current_user),
       drafts: Work.with_state(:first_draft).where(depositor: current_user)
     )
+
+    @presenter.work_stats = StatBuilder.build_stats if user_with_groups.administrator?
```

Comment:
This *feels* like a policy change but I don't feel strongly about whether that means this enforcement ought to move elsewhere.

Reply:
Yeah, I had considered this, but I wasn't convinced that we need a `StatsPolicy#show?` just yet.  If this gets more complicated, we could.